code/modules/spacepods/equipment.dm

```diff
@@ -40,8 +41,6 @@
 			playsound(src, fire_sound, 50, 1)
 			projone.dumbfire(my_atom.dir)
 			projtwo.dumbfire(my_atom.dir)
-		sleep(2)
```

Comment:
This sleep is rather important, it prevents the lasers from riding the same tiles